Automattic/wp-calypso
# Email: Create Titan control panel login component

#### Changes proposed in this Pull Request

This component will be shown when users want to manage their Titan Mail account, allowing them to log in to the control panel.

#### Testing instructions

* Inspect the code
* Render the component by hacking the code and verify the login button works as expected
